generic/tests/netperf.py

```diff
@@ -373,7 +386,7 @@ def start_test(server, server_ctl, host, clients, resultsdir, test_duration=60,
     base = params.get("format_base", "12")
     fbase = params.get("format_fbase", "2")
 
-    output = ssh_cmd(host, "mpstat 1 1 |grep CPU")
+    output = decode_to_text(ssh_cmd(host, "mpstat 1 1 |grep CPU"))
```

Comment:
Could you try to use ssh_cmd(host, "mpstat 1 1 | grep CPU").decode()?
If this can works well then you do not need to import decode_to_text.
The same as other places used decode_to_text().

Reply:
ssh_cmd calls process.system_output, so I directly decode process output to test in line 476 in ssh_cmd function. please help review, thanks